flink-streaming-java/src/main/java/org/apache/flink/streaming/api/transformations/PartitionTransformation.java

```diff
@@ -69,6 +88,15 @@ public PartitionTransformation(StreamTransformation<T> input, StreamPartitioner<
 		return partitioner;
 	}
 
+	/**
```

Comment:
I don't like these redundant descriptions to much, this can be simplified to:
```
/**
 * Returns the {@link ShuffleMode} of this {@link PartitionTransformation}.
 */
```

Reply:
Done